crates/bindings/src/service.rs

```diff
@@ -129,26 +131,64 @@ pub struct Channel {
 /// `log_dest_file_descriptor`. This file descriptor will be closed when the service is destroyed.
 /// If the `log_dest_file_descriptor` is <= 0, then logging will be disabled entirely for this
 /// service. Each service must use a unique `log_dest_file_descriptor` to avoid interleaved logs
-/// making the JSON output unparseable.
+/// making the JSON output unparsable.
 #[inline]
-pub fn create<S: Service>(log_level_filter: i32, log_dest_file_descriptor: i32) -> *mut Channel {
-    // Use service creation as a common entry hook through which we can install global tracing and
-    // logging. The global subscriber that's initialized here will only be used as a fallback for
-    // logs that are produced from other threads, so this is really just a bit of insurance to make
-    // sure we don't miss anything important.
-    crate::logging::setup_env_tracing();
-
-    // Now initialize the subscriber that will forward all the logs for this service to the
-    // `log_dest_file_descriptor`, if logging is enabled. If disabled, then we'll set a no-op
-    // subscriber.
-    let subscriber = if log_dest_file_descriptor > 0 {
-        new_thread_local_subscriber(log_level_filter, unsafe {
-            FileWriter::new(log_dest_file_descriptor)
-        })
+pub fn create<S: Service>(log_level: i32, log_dest_fd: i32) -> *mut Channel {
+    // Use service creation as a common entry hook through which we can install global tracing.
+    // The global subscriber initialized here will only be used as a fallback for events that
+    // are produced from other threads (of which there should be none), so this is really just
```

Comment:
It ought to be fairly obvious, because they'll show up in application logs (grafana)  rather than `ops/$tenant/logs`.

(This change also removes tenant task logs from the application log stream)